test/unit/lib/plugins/aws/package/compile/events/websockets/lib/stage.test.js

```diff
@@ -191,5 +191,139 @@ describe('#compileStage()', () => {
         ).to.equal(true);
       });
     });
+
+    it('should set a specific log Format if provider has format', () => {
```

Comment:
Let's not use the old-style tests - we're slowly moving away from them and for all new funcitonalities we're using `runServerless`-based tests - please see some examples in `websockets/index.test.js` at the end of the file.